Vates/VatesAPI/src/vtkSplatterPlotFactory.cpp

```diff
@@ -667,13 +654,11 @@ void vtkSplatterPlotFactory::setMetadata(vtkFieldData *fieldData,
 
   // Create a new field data array
   MetadataToFieldData convertMtoF;
-  vtkFieldData *outputFD = vtkFieldData::New();
+    vtkNew<vtkFieldData> outputFD;
   outputFD->ShallowCopy(fieldData);
-  convertMtoF(outputFD, xmlString, XMLDefinitions::metaDataId().c_str());
-  convertMtoF(outputFD, jsonString,
-              m_vatesConfigurations->getMetadataIdJson().c_str());
-  dataSet->SetFieldData(outputFD);
-  outputFD->Delete();
+    convertMtoF(outputFD.GetPointer(), xmlString, XMLDefinitions::metaDataId().c_str());
+    convertMtoF(outputFD.GetPointer(), jsonString, m_vatesConfigurations->getMetadataIdJson().c_str());
+    dataSet->SetFieldData(outputFD.GetPointer());
```

Comment:
Indentation in the whole file looks funny. Is that just a github display glitch, or did clang-format mess up?